frontend/src/components/Documentation.vue

```diff
@@ -673,48 +673,48 @@ export default {
         {
           name: 'Map Viewer',
           icon: 'fa-map-o',
-          link: '#Map-Viewer',
+          link: '#map-viewer',
           subsections: [
-            { name: '2D Viewer', link: '#2D-Viewer' },
-            { name: '3D Viewer', link: '#3D-Viewer' },
-            { name: 'Data Overlay', link: '#Data-overlay' },
+            { name: '2D Viewer', link: '#2d-viewer' },
+            { name: '3D Viewer', link: '#3d-viewer' },
+            { name: 'Data Overlay', link: '#data-overlay' },
           ],
         },
         {
           name: 'Interaction Partners',
           icon: 'fa-connectdevelop',
-          link: '#Interaction-Partners',
+          link: '#interaction-partners',
           subsections: [
-            { name: 'Export graph', link: '#Interaction-Partners-Export' },
-            { name: 'Highlights', link: '#Interaction-Partners-Highlights' },
-            { name: 'Reactions table', link: '#Interaction-Partners-Reactions' },
+            { name: 'Export graph', link: '#interaction-partners-export' },
+            { name: 'Highlights', link: '#interaction-partners-highlights' },
+            { name: 'Reactions table', link: '#interaction-partners-reactions' },
           ],
         },
         {
           name: 'Search',
           icon: 'fa-search',
-          link: '#Search',
+          link: '#search',
```

Comment:
For some reason, when I click on this from the ToC, it does not jump to the right section. It works correctly when refreshing the page though with `/documentation#search`. I tested on Chrome, Safari, and Firefox on macOS. Not a big problem though but it's an interesting behavior. All of the other links in the ToC seem to work. Wondering if it's something related specifically for the id `search`. The prod server with `#Search` also seems to work.

Reply:
Maybe because of caching? When I tested on all three Browsers it jumped to the right place.

Reply:
Follow up on this. It is because we already have an element that uses the same id.

<img width="1413" alt="image" src="https://user-images.githubusercontent.com/423498/157448902-ebde9836-f1ae-486e-b662-049749c70695.png">

Reply:
Well spotted! I renamed it to `searching` in e9a36db. Hope you all find that name ok.

Reply:
Looks great, thanks!